collections/_drafts/agent-forwarding.md

```diff
@@ -1,21 +1,18 @@
 ---
 title: OpenINF Next-Gen Guidance on Agent Forwarding
 category: contributing
-authors: @OpenINF/wg-a-team
-published: false
-permalink:
-layout:
+permalink: /docs/dev/internals/contributing/agent-forwarding/
 relevant_urls:
-- https://github.com/OpenINF/docker-fisher/issues/5 # psusan + auth awkwardness
-- https://dev.gnupg.org/T3883 # Win32-OpenSSH support for gpg-agent's ssh-agent
+  - https://github.com/OpenINF/docker-fisher/issues/5 # psusan + auth awkwardness
+  - https://dev.gnupg.org/T3883 # Win32-OpenSSH support for gpg-agent's ssh-agent
 ---
 
-To be able to sign any potential Git commits or tags you may make using your Git
-client, some additional setup may be necessary. This guide will be especially
-relevant if you develop inside a container we have provided. There will likely
-be snafus to overcome before connecting to the container, as we intend.
+Additional setup procedures may be necessary for the chosen few who hold core
+OpenINF membership to sign any potential Git commits or tags. This guide will be
+especially relevant for those developing inside the container provided as there
+will likely be snafus to overcome before connecting to the devcontainer.
 
-## 1.1    Connecting to GitHub with SSH
+## Connecting to GitHub with SSH
 
```

Comment:
i believe much of this is from the gh docs, but it needs to be rephrased and incorporated into this document